### ISSUE TITLE
About  evaluate.py

### ISSUE THREAD

**Issue by wtma1999** [Vaidehi99/InfoDeletionAttacks#6]
Great work, I'm following along and trying to reproduce your work, but facing some issues.
For three different attack strategies, you used three evaluate.py ( evaluate_parap_attack.py...) files, but after comparison, I found that the main parts are the same. I want to confirm with you whether the special feature of each file is in get_metrics(), which is used to reflect the three Attack strategy.

Looking forward to hearing from you!

**Comment by Vaidehi99** [comment]
Yes, they are quite similar scripts and have differences only in the way the attack is evaluated
